packages/terra-site/src/examples/table/NoStripedTable.jsx

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import Table from 'terra-table';
+
+const NoStripedTable = () => (
+  <Table isStriped={false}>
+    <Table.Header>
+      <Table.HeaderContent content={'Column Heading 1'} key={1} minWidth={'small'} />
+      <Table.HeaderContent content={'Column Heading 2'} key={2} minWidth={'medium'} />
+      <Table.HeaderContent content={'Column Heading 3'} key={3} minWidth={'large'} />
+    </Table.Header>
+    <Table.Rows>
+      <Table.Row key={0}>
```

Comment:
Totally agree on that. For this example, it feels too closely like it's setting the key equal to the value of it's array index, which also is not a good practice for assigning keys to react components. I feel like a developer would look at this example, and think that it could use the map function for example, and assign the key equal to the numerical index of that given element.